application/app/models/download_file.rb

```diff
@@ -57,13 +60,18 @@ def save
     return false unless valid?
 
     FileUtils.mkdir_p(Project.files_directory(project_id))
-    filename = self.class.filename_by_ids(project_id, id)
+    filename = self.class.metadata_path(project_id, id)
     File.open(filename, "w") do |file|
       file.write(to_hash.deep_stringify_keys.to_yaml)
     end
     true
   end
 
+  def destroy
+    file_path = self.class.metadata_path(project_id, id)
+    FileUtils.rm(file_path)
+  end
```

Comment:
Should we implement soft delete?

Reply:
Same as before - will add this to the previous story for projects

Reply:
https://github.com/hmdc/DevOpsProjects/issues/1739